packages/grid/data-grid/src/DataGrid.test.tsx

```diff
@@ -177,4 +177,122 @@ describe('<DataGrid />', () => {
       );
     });
   });
+
+  describe('column width', () => {
+    before(function beforeHook() {
+      if (/jsdom/.test(window.navigator.userAgent)) {
+        // Need layouting
+        this.skip();
+      }
+    });
+
+    it('should set the columns width to 100px by default', () => {
+      const rows = [
+        {
+          id: 1,
+          username: 'John Doe',
+          age: 30,
+        },
+      ];
+
+      const columns = [
+        {
+          field: 'id',
+        },
+        {
+          field: 'name',
+        },
+        {
+          field: 'age',
+        },
+      ];
+
+      render(
+        <div style={{ width: 300, height: 300 }}>
+          <DataGrid columns={columns} rows={rows} />
+        </div>,
+      );
+
+      const DOMColumns = document.querySelectorAll('.MuiDataGrid-colCell');
```

Comment:
Better use the accessibility features of the grid :)

```suggestion
      const DOMColumns = getAllByRole('columnheader');
```

Reply:
done